tools/tick-cluster.js

```diff
@@ -493,6 +498,19 @@ function killAllProcs() {
     }
 }
 
+function terminateProc(count) {
+    var processesToKill = _.chain(procs)
+        .filter(function (proc) { return !proc.killed && !proc.suspended; })
+        .sampleSize(+count)
+        .value();
+
+    _.each(processesToKill, function kill(proc) {
+        logMsg(proc.port, color.green('pid ' + proc.pid) + color.red(' randomly selected for termination'));
+        process.kill(proc.proc.pid, 'SIGTERM');
+        proc.killed = Date.now();
```

Comment:
I think we need to set a timeout a couple of seconds to verify that the application has been shutdown because an application can consume the `SIGTERM` without actually quitting. If the application did not stop after that timeout is should be send `SIGKILL`.